### ISSUE TITLE
om version 3.2.0 or up failed in 'om staged-config', got 404 error

### ISSUE THREAD

**Issue by jli1972** [pivotal-cf/om#419]
could not execute "staged-config": request failed: unexpected response:
HTTP/1.1 404 Not Found
Connection: keep-alive
Content-Type: text/html; charset=UTF-8
Date: Thu, 29 Aug 2019 04:50:37 GMT
Server: nginx/1.10.3 (Ubuntu)
Strict-Transport-Security: max-age=15552000; includeSubDomains
X-Request-Id: df5b362a-df86-46e0-84c3-c8f0312a793a
X-Runtime: 0.001158
Content-Length: 0

**Comment by weaverb** [comment]
We are also seeing this error with 3.2.1

**Comment by kcboyle** [comment]
Hi @weaverb ! 
What version of Ops Manager are you on? We ran a test of this on 2.3.25 and got the following response:
```
GET /api/v0/staged/products/cf-79fba6887e8c29375eb7/syslog_configuration HTTP/1.1
Content-Type: application/json


HTTP/1.1 422 Unprocessable Entity
Transfer-Encoding: chunked
Cache-Control: no-cache, no-store
Connection: keep-alive
Content-Type: application/json; charset=utf-8
Date: Thu, 29 Aug 2019 22:18:18 GMT
Expires: Fri, 01 Jan 1990 00:00:00 GMT
Pragma: no-cache
Server: Ops Manager
Strict-Transport-Security: max-age=15552000; includeSubDomains
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-Request-Id: 657a651c-7ae0-44db-9b45-aa1884199c4a
X-Runtime: 0.466983
X-Xss-Protection: 1; mode=block

114
{"errors":{"syslog_configuration":["This product does not support the Ops Manager consistent syslog configuration feature. If the product supports custom syslog configuration, those properties can be set via the /api/v0/staged/products/:product_guid/properties endpoint.\n"]}}
0

```

however, `om` takes this response and the error and will continue without actual failure (the only way we could check for the existence of the endpoint was to hit it. If the endpoint is not found, we continue without error) during the staged-config, and we were able to get the configuration successfully. 

We are going to try this with the dataflow tile to see if there is something specific with that tile we may be missing.

**Comment by kcboyle** [comment]
this will be fixed in `om` 3.2.2 later today. Thank you for bringing this to our attention!

**Comment by jli1972** [comment]
confirm om-3.2.2 fixed this issue. 

Thanks!
